packages/core/utils/src/validate/validators.ts

```diff
@@ -51,6 +51,12 @@ const defaultValidateFilters = curry((schema: Model, filters: unknown) => {
     // empty objects
     traverseQueryFilters(
       ({ key, value }) => {
+        // ID is not an attribute per se, so we need to make
+        // an extra check to ensure we're not validating it
+        if (key === 'id') {
+          return;
+        }
+
```

Comment:
It looks like you just updated the src, but there is no tests against it, nor external urls descript how this comes.

Reply:
Will there be unit test agains such modification?
Will there be an issue url amended into the comment?

Reply:
You're right, I will be adding tests for this, but I think there is a second issue that needs to be solved as well before the issue is actually fixed.